setup.py

```diff
@@ -73,7 +73,7 @@ def run(self):
             arch = "x86_64"
             try:
                 sp.check_call(f"src/MotifSampler/MotifSampler_{arch}")
-            except sp.CalledProcessError as e:
```

Comment:
Any downside to changing this to:
```
except (sp.CalledProcessError, OSError) as e:
```

Reply:
Might as well give it a shot and see if anyone else complains. Theoretically possible that there would be other errors...I'll push another commit.